notebooks/02.06-Boolean-Arrays-and-Masks.ipynb

```diff
@@ -8,28 +8,15 @@
     "<img align=\"left\" style=\"padding-right:10px;\" src=\"figures/PDSH-cover-small.png\">\n",
```

Comment:
<p>5th comment - Lots of print stmts</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amit1rrr/PythonDataScienceHandbook/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>